src/js/components/stock-movement-wizard/EditPage.jsx

```diff
@@ -366,7 +364,7 @@ class EditItemsPage extends Component {
     this.props.showSpinner();
     this.reviseRequisitionItems(formValues)
       .then(() => {
-        if (this.state.statusCode === 'VERIFYING' || this.state.redoAutopick) {
+        if (this.state.redoAutopick) {
```

Comment:
you need to remove this "if", because after this change transition to next step will be done only if redoAutopick is true